.babelrc

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Rather than having this file (`.babelrc`) and `.flowconfig`, could we have these in a separate repository and import here? Would be a lot better, otherwise this is gonna be duplicated everywhere.

Reply:
I think ideally we should move these into AEgir